### PR TITLE
Update publishing-to-app-store.md

### PR DESCRIPTION
**Pull request by himesh-cm**
clarified how the publishing will work with wfe for iOS.

### CODE REVIEW

File: content/flutter-publishing/publishing-to-app-store.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ Alternatively, if you wish to submit an already uploaded build for review in App
 3. To start the submission process, click **Prepare for Submission**.
 4. Check that your app metadata is up to date, and once everything is ready, click the **Submit for Review** button.
 
-Note that the application must be manually released to the App Store once Apple approves the release.
+Please note that when using the workflow editor, the application will be manually released to the App Store by default once it is approved by Apple. However, you have the option to set up automatic publishing by making the necessary changes in your `Apple developer account > Apps > Distribution settings`.
```
Comment (icarusdust) [inline content/flutter-publishing/publishing-to-app-store.md:72]: Two things here:

1. Do you mean that apps will need to be released manually by users after Apple approves their submission
2. Where exactly is Apple developer account > Apps > Distribution settings` and what settings you are referring to ?

Comment (himesh-cm) [reply content/flutter-publishing/publishing-to-app-store.md]: @icarusdust can you please review it now, i have made the required changes to the path.

Comment (icarusdust) [reply content/flutter-publishing/publishing-to-app-store.md]: I suggest you explain what release methods exist and then, where these methods can be configured in App Store Connect. By explaining it, you can highlight what the default behaviour is.

Comment (himesh-cm) [reply content/flutter-publishing/publishing-to-app-store.md]: @icarusdust, i have clarified what it means, please review

File: content/flutter-publishing/publishing-to-app-store.md [hunk 2]
```diff
@@ -69,7 +69,29 @@ Alternatively, if you wish to submit an already uploaded build for review in App
 3. To start the submission process, click **Prepare for Submission**.
 4. Check that your app metadata is up to date, and once everything is ready, click the **Submit for Review** button.
 
-Note that the application must be manually released to the App Store once Apple approves the release.
+When using the workflow editor, developers have a few different methods to choose from for publishing their app on Apple's App Store once it has been approved by Apple. Each method caters to different strategies and needs. Here’s a breakdown of the release options available and how you can configure them in App Store Connect:
```
Comment (icarusdust) [inline content/flutter-publishing/publishing-to-app-store.md:72]: Would be better to explain what the default behaviour is by WFE

Comment (himesh-cm) [reply content/flutter-publishing/publishing-to-app-store.md]: Added, please review.

**Review by icarusdust — APPROVED** [review]
lgtm

**Pull request merged**